templates_ark/example_pixel_clustering.ipynb

```diff
@@ -632,13 +632,18 @@
     "    )\n",
```

Comment:
<p>This whole workflow needs to be batched. The user will first generate the masks, then save them. If these are batched separately, then the end result will still be a single variable holding the output of all 1200 FOVs, which is what we want to avoid. </p><p> </p><p>Instead, there needs to be a single function that generates the masks, and saves the masks, and this should have a batch size so that everything can be processed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>